waiter/integration/waiter/websocket_integration_test.clj

```diff
@@ -153,6 +160,62 @@
         (finally
           (delete-service waiter-url waiter-headers))))))
 
+(deftest ^:parallel ^:integration-slow test-fail-to-stream-large-responses
+  (testing-using-waiter-url
+    (let [^WebSocketClient websocket-client (websocket-client-factory)
+          waiter-settings (waiter-settings waiter-url)
+          {:keys [ws-max-binary-message-size ws-max-text-message-size]} (:websocket-config waiter-settings)
+          ws-max-binary-message-size' (+ 2048 ws-max-binary-message-size)
```

Comment:
We need kitchen to support larger than Waiter's configured max (adding the 2048 to waiter config).
We then need kitchen to send a message larger than Waiter's configured max but smaller than what kitchen can handle (adding to 1024 (i.e. < 2048) to waiter config).